lib/publiccloud/gce.pm

```diff
@@ -51,11 +51,17 @@ sub upload_img {
     # See https://cloud.google.com/sdk/gcloud/reference/compute/images/create for a list of available features
     # SEV_CAPABLE is added because all images from 15-SP2 onwards support SEV
     my $guest_os_features = get_var('PUBLIC_CLOUD_GCE_UPLOAD_GUEST_FEATURES', 'MULTI_IP_SUBNET,UEFI_COMPATIBLE,VIRTIO_SCSI_MULTIQUEUE,SEV_CAPABLE');
+    my $arch = get_var('PUBLIC_CLOUD_ARCH', '');
```

Comment:
There are already places where we assume that the default architecture is `x86_64` so you can do:
```suggestion
    my $arch = get_var('PUBLIC_CLOUD_ARCH', 'x86_64');
```

Reply:
But this would change the current workflow, where we don't set the architecture.

So, except for cosmetic changes - why would this be better?

Reply:
I also thought what Pavel suggesting but than I concluded exactly the same what Felix replying :)

Reply:
I don't really get how the 'workflouw' would be changed? Everywhere is by default expected `x86_64` … why is this suddenly different?

Reply:
Probably no. We can leave this for another refactoring.

But it is adding 6 lines and one condition vs. one line and zero conditions.

Reply:
> Probably no. We can leave this for another refactoring.
> 
> But it is adding 6 lines and one condition vs. one line and zero conditions.

But all other modifications are still needed, because we haven't used the `--architecture` parameter before. That's exactly why I don't see why this change would make any difference.

We haven't been using `PUBLIC_CLOUD_ARCH` in this test run at all before, it got introduced in this PR.